### DISCUSSION TITLE
Tracking amazon fees

### DISCUSSION THREAD

**Discussion by ianrussel** [amzn/selling-partner-api-models#3257] [General]
Hi, is anyone here tried to track amazon fees? Specifically Subscription and fbaremovalfee. I tried the financial events API and reports API, seem both incomplete. Below is a sample response from FinancialEvents endpoint. The response inside ServiceFeeEventList doesn't contain other helpful information like `date`.

         ` "ServiceFeeEventList": [
            {
                "AmazonOrderId": "T630nVKh2j",
                "FeeList": [
                    {
                        "FeeType": "FBARemovalFee",
                        "FeeAmount": {
                            "CurrencyCode": "EUR",
                            "CurrencyAmount": -0.25
                        }
                    }
                ]
            },
            {
                "AmazonOrderId": "T630nVKh2j",
                "FeeList": [
                    {
                        "FeeType": "FBARemovalFee",
                        "FeeAmount": {
                            "CurrencyCode": "EUR",
                            "CurrencyAmount": -0.25
                        }
                    }
                ]
            },
            {
                "FeeList": [
                    {
                        "FeeType": "Subscription",
                        "FeeAmount": {
                            "CurrencyCode": "EUR",
                            "CurrencyAmount": -39
                        }
                    }
                ]
            }
        ],
        "AdjustmentEventList": [
            {
                "AdjustmentType": "CommissionCorrection",
                "PostedDate": "2022-01-19T02:45:25+00:00",
                "AdjustmentAmount": {
                    "CurrencyCode": "EUR",
                    "CurrencyAmount": 134.59
                }
            },`
`

I need to show the amazonfees into the customer portal filter by date. Any ideas where to get this information? 

Regards

**Comment by takeno-dev** [comment]
I think that is the problem too.
I want to get these Date so I am running the process in periods of 1 day each. It is very annoying.
